python/ray/tests/test_multi_node.py

```diff
@@ -327,6 +327,53 @@ def test_calling_start_ray_head():
             ["ray", "start", "--head", "--redis-address", "127.0.0.1:6379"])
     subprocess.Popen(["ray", "stop"]).wait()
 
+    # Test --block. Killing any child process should cause the command to exit.
+    blocked = subprocess.Popen(["ray", "start", "--head", "--block"])
+    blocked.poll()
+    assert blocked.returncode is None
```

Comment:
Consider removing this assert as I don't think it is meaningful. Even without the `--block` it will probably pass because the `ray start` command takes a second or two to finish running, and `Popen` returns right away.